### PR TITLE
Enable support for 3scale/toolbox support for ppc64le(tested on UBI9)

### PR DESCRIPTION
**Pull request by shubham-bhagwat0**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Have you validated script on UBI 9 container
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [x] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: 3/3scale_toolbox/3scale-toolbox_ubi_9.3.sh [hunk 1]
```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+# ---------------------------------------------------------------------
+#
+# Package       : 3scale/toolbox
+# Version       : 2.14.1
+# Source repo   : https://github.com/3scale/3scale_toolbox
+# Tested on     : UBI 9.3 (UBI9)
```
Comment (AbhiIBM) [inline 3/3scale_toolbox/3scale-toolbox_ubi_9.3.sh:7]: Tested on value should be "UBI:9.3" else the script will fail.

File: 3/3scale_toolbox/3scale-toolbox_ubi_9.3.sh [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+ 
+#install 3scale/3scale_toolbox
+gem install 3scale_toolbox
+gem install rails
+bundle install
+gem install racc
+ 
+# Rake install
+ 
+bundle exec rake install
```
Comment (AbhiIBM) [inline 3/3scale_toolbox/3scale-toolbox_ubi_9.3.sh:101]: Add proper "if fi" here for building package.

File: 3/3scale_toolbox/3scale-toolbox_ubi_9.3.sh [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+ 
+# Running Unit Test
+ 
+if ! bundle exec rake spec:unit; then
+	echo "------------------$PACKAGE_NAME:install_success_but_test_fails---------------------"
+	echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | GitHub | Fail |  Install_success_but_test_Fails"
+	exit 1
```
Comment (AbhiIBM) [inline 3/3scale_toolbox/3scale-toolbox_ubi_9.3.sh:108]: Exit value should be 2 here.

**Review by shubham-bhagwat0 — COMMENTED** [review]
changes made, fixed suggested issues

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**